docker/run-apache2.sh

```diff
@@ -48,11 +49,19 @@ a2ensite spinnaker
 
 cp docker/ports.conf.gen ports.conf
 
-sed -ie "s/{%DECK_HOST%}/$_DECK_HOST/g" ports.conf
-sed -ie "s/{%DECK_PORT%}/$_DECK_PORT/g" ports.conf
-
+sed -ie "s|{%DECK_HOST%}|$_DECK_HOST|g" ports.conf
```

Comment:
I changed these as well to match the other use of `|` as a the sed expression delimiter since the deck host could contain `/` which would break the sed expression.